platform/binance/api_test.go

```diff
@@ -468,28 +468,28 @@ type testToken struct {
 func TestNormalizeToken(t *testing.T) {
 	testingTokens := []testToken{
 		{
-			name:        "token",
+			name:        "Aeron",
 			apiResponse: myToken,
 			tokens:      tokenList,
 			expected:    tokenDst,
 			ok:          true,
 		},
 		{
-			name:        "token",
+			name:        "Binance Chain Native Token",
```

Comment:
what does the name mean here? I think name should contain the unit test name correct?

Reply:
fixed